akka-sample-sharding-scala/src/main/scala/sample/sharding/Message.scala

```diff
@@ -1,9 +1,9 @@
 /**
-  * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
-  */
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
 package sample.sharding
 
 /**
-  * Marker interface for actor messages that are serialized.
-  */
+ * Marker interface for actor messages that are serialized.
+ */
 trait Message
```

Comment:
Change it to the recommended `JsonSerializable` or `CborSerializable` ( https://doc.akka.io/docs/akka/2.6/serialization-jackson.html#usage )

Reply:
This is already configured: see application.conf:

```
serialization-bindings {
      "sample.sharding.Message" = jackson-cbor
    }
```

Reply:
Yep, and the recommendation is to call it JsonSerializable or CborSerializable since the entire reason for having the marker trait is to tie it in to serialization.